src/main/java/org/killbill/billing/catalog/api/CatalogUserApi.java

```diff
@@ -33,11 +33,12 @@
     /**
      * Retrieves the multi versioned catalog.
      *
-     * @param catalogName the name of the catalog
-     * @param context     the user context that specifies the tenant information
+     * @param catalogName        the name of the catalog
+     * @param catalogDateVersion optional date filter to retrieve a specific version
+     * @param context            the user context that specifies the tenant information
      * @return the {@code Catalog}
      */
-    Catalog getCatalog(String catalogName, TenantContext context) throws CatalogApiException;
+    VersionedCatalog<? extends StaticCatalog> getCatalog(String catalogName, DateTime catalogDateVersion, TenantContext context) throws CatalogApiException;
```

Comment:
What is the use case for this `catalogDateVersion ` filter - should we use it to return one version or set of version after such date?

Reply:
It performs an exact match (mainly used by Kaui).